.eslintrc.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   root: true,
+  parser: 'babel-eslint',
```

Comment:
Lets roll this back, and just rely on promise chaining.  I know that is annoying, but its pretty important to ensure that this library doesn't *require* babel-polyfill and whatnot...